### ISSUE TITLE
Incorrect name is provided for maow link as logo: A11y_Maow_Home Page_Name Role Value.

### ISSUE THREAD

**Issue by AnKushSingh05** [microsoft/moaw#116]
### GitHub Tags
#A11yTCS; #A11ySev2; #ChromiumEdge; #WCAG4.1.2; #Win11; #GH_Maow_Web_Mar24; #Maow; #Name Role Value; #AILimited: #DesktopWeb; #A11yMAS;

### Environment Details:
Application Name: Maow
#URL: https://moaw.dev/
OS: Windows 11 Enterprise 23H2 (OS build 22631.3296)
Microsoft Edge: Version 122.0.2365.92 (Official build) (64-bit)

### Repro Steps:

1. Hit the URL https://moaw.dev/
2. Moaw screen will get open.
3. TAB till header section.
4. Verify that all the element of "Header" is accessible and MAS compliant.
### Actual Result:
Incorrect name is provided for maow link as logo
### Expected Result:
Correct name should be provided for link as maow.
### User Impact:
Screen reader users will get the wrong information if incorrect name is provided.
### Attachment:
![Bug_116](https://github.com/microsoft/moaw/assets/124244478/31c778bb-7a83-4306-9a76-3625ed29b69b)

**Comment by sinedied** [comment]
@AnKushSingh05 this one has been fixed, could you validate it?

**Comment by AnKushSingh05** [comment]
Verified the issue in below environment
https://moaw.dev/
Issue is fixed. Hence closing the bug.

![Screenshot (1949)](https://github.com/microsoft/moaw/assets/124244478/8925fe36-b8fb-4e4d-90e8-128d35f42263)
